### PR TITLE
Fixed TramCommonCommandFactory.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: eventuate-tram-micronaut-commands/src/main/java/io/eventuate/tram/micronaut/commands/common/TramCommandCommonFactory.java [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 import io.eventuate.tram.commands.common.CommandNameMapping;
 import io.eventuate.tram.commands.common.DefaultCommandNameMapping;
+import io.micronaut.context.annotation.Factory;
 import io.micronaut.context.annotation.Requires;
 
 import javax.inject.Singleton;
 
+@Factory
 public class TramCommandCommonFactory {
```
Comment (cer) [inline eventuate-tram-micronaut-commands/src/main/java/io/eventuate/tram/micronaut/commands/common/TramCommandCommonFactory.java:11]: What about a test for this? This is.a.bug, right?

**Review by cer — CHANGES_REQUESTED** [review]

File: eventuate-tram-micronaut-commands/src/main/java/io/eventuate/tram/micronaut/commands/common/TramCommandCommonFactory.java (hunk 1 above)
Comment (dartartem) [reply eventuate-tram-micronaut-commands/src/main/java/io/eventuate/tram/micronaut/commands/common/TramCommandCommonFactory.java:11]: Without factory annotation, it does not create bean.
Do we really need to test this?

Comment (cer) [reply eventuate-tram-micronaut-commands/src/main/java/io/eventuate/tram/micronaut/commands/common/TramCommandCommonFactory.java:11]: * There's some functionality in Tram Sagas that was broken by a bug in Tram.
* Yet the tests for Tram passed.
* Sounds like there is a missing test.

Comment (dartartem) [reply eventuate-tram-micronaut-commands/src/main/java/io/eventuate/tram/micronaut/commands/common/TramCommandCommonFactory.java:11]: ok, I will add

**Pull request merged**